### ISSUE TITLE
Feature Request: Instead of requiring two layers in a selection, when selecting one object, it would be good if "white" was a default second layer.

### ISSUE THREAD

**Issue by richardsison** [stark-contrast/stark-sketch-plugin#26]
When checking the accessibility of a colour, most times at a default I want to see if it's either accessible against white or black.

I thought this was an option with the previous version of Stark? I may be mistaken though.

Regardless, if instead of having two layers selected as a default, it would be good if "white" (or black) could be a "pseudo-selection" if only one layer is selected.

Does that make sense?

**Comment by SpacyRicochet** [comment]
Or analogous, some way of 'locking' a color to be the one to check against. For example, when you select a single layer, you could do `⇧+⌥+⌘+P` and lock that specific fill.
